sqs/variables.tf

```diff
@@ -50,6 +50,14 @@ variable "create_cloudwatch_alarm" {
   default = true
 }
 
+variable "encryption_type" {
+  default = "kms"
```

Comment:
What's the least breaking change to set this default? Defaulting to `kms` will break any use of this module where no kms key has been passed. But defaulting to `sse` will break anyone using KMS.